app/Console/Commands/LdapSync.php

```diff
@@ -51,7 +54,10 @@ public function handle()
         $ldap_result_active_flag = Setting::getSettings()->ldap_active_flag_field;
         $ldap_result_emp_num = Setting::getSettings()->ldap_emp_num;
         $ldap_result_email = Setting::getSettings()->ldap_email;
-
+        
+        $permissions = config('permissions');
+		$permissions_array = Helper::selectedPermissionsArray($permissions, Input::old('permissions', array()));
```

Comment:
We should be using the `old()` helper here, instead of the `Input::` facade.

Also where is this old data coming from? This is a console command, so there isn't any `request` data being passed.

Reply:
I have literally no knowledge of PHP, although i still understand what is happening in the code and what you mean. I just had this problem and i tried to fix it by looking up, how a user was normally created and from there it was just copy-paste with some testing until i found this to be working. But your totally right, that the old input is not needed. I even think, that the "use Input;" line is unnecessary then aswell.